source/includes/extracts-update-operations-base.yaml

```
@@ -0,0 +1,7 @@
+ref: _update-operation-empty-operand-expressions
+content: |
+  Starting in MongoDB 5.0, :binary:`~bin.mongod` no longer raises an
+  error when you use an update operator like :update:`${{operator}}`
+  with an empty operand expression ( ``{ }`` ). An empty update results
+  in no changes and no :term:`oplog` entry is created (i.e. the
```

Our [style guide](https://docs.mongodb.com/meta/style-guide/terminology/general-term-guidelines/avoid-obscure-words/) discourages the use of "i.e.", since it often does not translate well for a global audience.

```suggestion
  in no changes and no :term:`oplog` entry is created (meaning that the
```